Polyfills/Window/Source/Window.cpp

```diff
@@ -38,6 +37,11 @@ namespace Babylon::Polyfills::Internal
             global.Set(JS_SET_TIMEOUT_NAME, Napi::Function::New(env, &Window::SetTimeout, JS_SET_TIMEOUT_NAME, Window::Unwrap(jsWindow)));
         }
 
+        if (global.Get(JS_CLEAR_TIMEOUT_NAME).IsUndefined())
```

Comment:
We should probably check for both `setTimeout` and `clearTimeout` together. If there is an exisitng implementation of `setTimeout` but not `clearTimeout` or vice versa, this code is going to polyfill one of the functions and that won't work at all.